src/module/apps/characterImport/CharacterInfoUpdater.js

```diff
@@ -90,6 +90,10 @@ export class CharacterInfoUpdater {
         this.importInitiative(clonedActorSource.system, chummerChar);
         this.importSkills(clonedActorSource.system, chummerChar);
 
+        if(chummerChar.critterpowers?.critterpower) {
+            clonedActorSource.system.is_critter = true;
```

Comment:
This is a new system field? What is it's differentation to the actor.type === 'critter' field?

Reply:
No it is an existing field that controls if critter powers tab is shown